packages/installer.vm/tools/chocolateyinstall.ps1

```diff
@@ -32,14 +32,7 @@ try {
     }
     VM-Write-Log "INFO" "Packages installation complete"
 
-    ## Configure taskbar with custom Start Layout if it exists.
-    $customLayout = Join-Path ${Env:VM_COMMON_DIR} "CustomStartLayout.xml"
-    if (Test-Path $customLayout) {
-        Import-StartLayout -LayoutPath $customLayout -MountPath "C:\"
-        Stop-Process -Name explorer -Force  # This restarts the explorer process so that the new taskbar is displayed.
-    } else {
-        VM-Write-Log "WARN" "CustomStartLayout.xml missing. No items will be pinned to the taskbar."
-    }
+    Stop-Process -Name explorer -Force  # This restarts the explorer process so that the new taskbar is displayed.
```

Comment:
[nitpick] As we are not setting the taskbar here anymore, this code looks out of place now. What about adding this code to the `VM-Refresh-Desktop` function or moving this line after the call to `VM-Refresh-Desktop`?